streams/src/test/java/org/apache/kafka/streams/integration/KStreamAggregationDedupIntegrationTest.java

```diff
@@ -157,7 +157,7 @@ public void shouldReduceWindowed(final TestInfo testInfo) throws Exception {
         produceMessages(secondBatchTimestamp);
 
         groupedStream
-            .windowedBy(TimeWindows.of(ofMillis(500L)))
+            .windowedBy(TimeWindows.ofSizeAndGrace(ofMillis(500L), ofMinutes(1L)))
```

Comment:
As above -- why is this change ok, w/o the need to change the test code, too?

Reply:
In this test, we create topic with 3 partitions,  https://github.com/apache/kafka/blob/9ccc33da8af2c506647620a6a6f1795030328de1/streams/src/test/java/org/apache/kafka/streams/integration/KStreamAggregationDedupIntegrationTest.java#L241
and records with secondBatchTimestamp may come after records with firstBatchTimestamp, so we need grace period here to avoid records being dropped.
https://github.com/apache/kafka/blob/9ccc33da8af2c506647620a6a6f1795030328de1/streams/src/test/java/org/apache/kafka/streams/integration/KStreamAggregationDedupIntegrationTest.java#L154-L157